internal/appsec/waf/waf.go

```diff
@@ -326,12 +333,8 @@ func goRunError(rc C.DDWAF_RET_CODE) error {
 }
 
 func getWAFVersion() string {
-	var v C.ddwaf_version
-	C.ddwaf_get_version(&v)
-	major := uint16(v.major)
-	minor := uint16(v.minor)
-	patch := uint16(v.patch)
-	return fmt.Sprintf("%d.%d.%d", major, minor, patch)
+	cversion := C.ddwaf_get_version() // static mem pointer returned - no need to free it
```

Comment:
I guess we could oneline this unless we'd rather keep it as is for debug purpose.